### ISSUE TITLE
Bug: Gluetun picks a random region for Surfshark

### ISSUE THREAD

**Issue by frepke** [qdm12/gluetun#593]
**Is this urgent?**: half

**Host OS** (approximate answer is fine too): Debian Buster

**CPU arch** or **device name**: amd64

**What VPN provider are you using**: Surfshark

**What are you using to run your container?**: Docker Compose

**What is the version of the program** (See the line at the top of your logs)

```
Running version latest built on 2021-08-27T19:15:11Z (commit 3863cc4)
```

**What's the problem** 🤔

Gluetun pick a random region for Surfshark
In the compose-file:
REGION=Netherlands Amsterdam
but connected to the US server (US server in latest log, but this is random)

**Share your logs... (careful to remove in example tokens)**

```log


today at 08:59:14 2021/08/28 08:59:14 INFO vpn: OpenVPN 2.5.2 x86_64-alpine-linux-musl [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [MH/PKTINFO] [AEAD] built on May  4 2021
today at 08:59:14 2021/08/28 08:59:14 INFO vpn: library versions: OpenSSL 1.1.1l  24 Aug 2021, LZO 2.10
today at 08:59:14 2021/08/28 08:59:14 WARN vpn: --ping should normally be used with --ping-restart or --ping-exit
today at 08:59:14 2021/08/28 08:59:14 INFO vpn: TCP/UDP: Preserving recently used remote address: [AF_INET]173.237.207.11:1194
today at 08:59:14 2021/08/28 08:59:14 INFO vpn: UDP link local: (not bound)
today at 08:59:14 2021/08/28 08:59:14 INFO vpn: UDP link remote: [AF_INET]173.237.207.11:1194
today at 08:59:14 2021/08/28 08:59:14 INFO vpn: [us-bos-v010.prod.surfshark.com] Peer Connection Initiated with [AF_INET]173.237.207.11:1194

```

**Comment by qdm12** [comment]
If this is urgent, use `CITY: Amsterdam` instead for now. It's my retro-compatibility layer for older `REGION` names that is not properly coded, I'm fixing it now but it's taking longer than I expected.

**Comment by qdm12** [comment]
Fixed in 29d92fd307eee291bc7da083c6fb9036a951d1a4 😉
